### PR TITLE
EU-84

### PR DESCRIPTION
**Pull request by DmytroLyt**
Created file gunicorn.conf.py and used uvicorn.workers.UvicornWorker with it to run a back-end service.

### CODE REVIEW

File: gunicorn.conf.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+# Debugging
+reload = False
+reload_engine = 'auto'
+spew = False
+check_config = False
+print_config = False
+
+
+# Server socket
+bind = '127.0.0.1:8000'
```
Comment (Kostiantyn-Salnykov) [inline gunicorn.conf.py]: You can setup bind dynamically through the settings managements (HOST, PORT) parameters.

File: gunicorn.conf.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+backlog = 2048
+
+
+# Worker processes
+workers = 4
+worker_class = 'uvicorn.workers.UvicornWorker'
+worker_connections = 1000
+max_requests = 0
```
Comment (Kostiantyn-Salnykov) [inline gunicorn.conf.py]: Set `max_requests` to 50-100 (reduce chance of memory leaks) in future.
Also add `max_requests_jitter` to 3-5.

**Review by Kostiantyn-Salnykov — CHANGES_REQUESTED** [review]

File: gunicorn.conf.py (hunk 1 above)
Comment (DmytroLyt) [reply gunicorn.conf.py]: Did it

File: gunicorn.conf.py (hunk 2 above)
Comment (DmytroLyt) [reply gunicorn.conf.py]: > Set `max_requests` to 50-100 (reduce chance of memory leaks) in future. Also add `max_requests_jitter` to 3-5.

Okay. I fixed all issues

**Pull request merged**